packages/firestore/src/lite-api/aggregate.ts

```diff
@@ -15,72 +15,61 @@
  * limitations under the License.
  */
 
-import { Value } from '../protos/firestore_proto_api';
-import { invokeRunAggregationQueryRpc } from '../remote/datastore';
-import { hardAssert } from '../util/assert';
+import { Query, queryEqual } from './reference';
 import { cast } from '../util/input_validation';
-
-import { getDatastore } from './components';
 import { Firestore } from './database';
-import { Query, queryEqual } from './reference';
+import { getDatastore } from './components';
 import { LiteUserDataWriter } from './reference_impl';
+import { invokeRunAggregationQueryRpc } from '../remote/datastore';
+import { hardAssert } from '../util/assert';
+import { Value } from '../protos/firestore_proto_api';
+import { deepEqual } from '@firebase/util';
 
-/**
- * An `AggregateQuery` computes some aggregation statistics from the result set of
- * a base `Query`.
- */
-export class AggregateQuery {
-  readonly type = 'AggregateQuery';
-  /**
-   * The query on which you called `countQuery` in order to get this `AggregateQuery`.
-   */
-  readonly query: Query<unknown>;
-
-  /** @hideconstructor */
-  constructor(query: Query<unknown>) {
-    this.query = query;
-  }
+export type DocumentFieldValue = any;
```

Comment:
This definition of `DocumentFieldValue` needs to be moved. There are a bunch of places in our code that define `DocumentData` mapping strings to `any`. These places all need to replace `any` with `DocumentFieldValue`, and define `DocumentFieldValue` immediately above it. Then, import `DocumentFieldValue` into this file.

Here are the places I could find that need to be updated:
- https://github.com/firebase/firebase-js-sdk/blob/98189d5b18bdcf0958c62dcf833f0c22be786214/repo-scripts/prune-dts/tests/firestore.input.d.ts#L914-L916
- https://github.com/firebase/firebase-js-sdk/blob/98189d5b18bdcf0958c62dcf833f0c22be786214/packages/firebase/compat/index.d.ts#L8230-L8234
- https://github.com/firebase/firebase-js-sdk/blob/98189d5b18bdcf0958c62dcf833f0c22be786214/packages/firestore-types/index.d.ts#L20
- https://github.com/firebase/firebase-js-sdk/blob/98189d5b18bdcf0958c62dcf833f0c22be786214/repo-scripts/prune-dts/tests/firestore.output.d.ts#L344-L350
- https://github.com/firebase/firebase-js-sdk/blob/98189d5b18bdcf0958c62dcf833f0c22be786214/packages/firestore/src/lite-api/reference.ts#L47-L51

Some of these places may be auto-generated. Also, in the places that require documentation, please put in some brief documentation, such as "A convenience type to indicate that the value is a Firestore value in a document."